azure/table_azure_security_center.go

```diff
@@ -0,0 +1,207 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/resources/mgmt/policy"
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/v1.0/security"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureSecurityCenter(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_security_center",
+		Description: "Azure Security Center",
+		List: &plugin.ListConfig{
+			Hydrate: listSecurityCenter,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "auto_provisioning",
+				Description: "Auto provisioning settings of the subscriptions.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "contact",
+				Description: "Security contact configurations for the subscription.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "policy",
+				Description: "Provides operations to assign policy definitions to a scope in your subscription.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "pricing",
+				Description: "Security pricing configuration in the resource group.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "setting",
+				Description: "Configuration settings for Azure Security Center.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromConstant("Security Center"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Azure standard columns
+			{
+				Name:        "subscription_id",
+				Description: ColumnDescriptionSubscription,
+				Type:        proto.ColumnType_STRING,
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listSecurityCenter(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Comment:
@bigdatasourav Should this be `listSecurityCenters`? Or if we can't get more than 1 security center, is there a different function name we should use?